packages/admin/admin-rte/src/core/translation/stateToHtml.ts

```diff
@@ -0,0 +1,39 @@
+import { EditorState } from "draft-js";
+import { stateToHTML } from "draft-js-export-html";
+
+import { IRteOptions } from "../Rte";
+
+export function stateToHtml({ editorState, options }: { editorState: EditorState; options: IRteOptions }) {
+    const contentState = editorState.getCurrentContent();
+
+    const customInlineStyleKeys = options?.customInlineStyles ? Object.keys(options.customInlineStyles) : [];
+
+    const inlineStyles: any = {
```

Comment:
nit: Is there an alternative to this any cast? E.g. no explicit typing at all?

Reply:
ecd75ec5305ce97c7f7f7d92255093b287cda412

![Screenshot 2024-02-20 at 14 08 39](https://github.com/vivid-planet/comet/assets/118435139/0d2fc250-ece2-4c14-9973-e68e1428a35a)